package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "csv-generate": "^2.2.2",
-    "csv-parse": "^3.1.2",
+    "csv-parse": "^3.1.3",
```

Comment:
I can put this into a separate PR, but this is so tiny that thought might not be worth the hassle.